packages/react-tabs/src/components/Pivot/Pivot.styles.ts

```diff
@@ -199,18 +199,27 @@ export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
         [`&[data-is-overflowing='true']`]: {
           display: 'none',
           selectors: {
-            [`~ * .${classNames.overflowMenuButton}`]: {
+            [`~ .${classNames.overflowMenuButton}`]: {
               visibility: 'visible',
+              position: 'relative',
             },
           },
         },
       },
     ],
     overflowMenuButton: [
       classNames.overflowMenuButton,
-      ...getLinkStyles(props, classNames),
       {
-        visibility: 'hidden',
+        // TODO after the extra span wrapper around the button is removed,
+        // remove the 'button&' and 'span&' selectors and combine the styles
+        ['button&']: [...getLinkStyles(props, classNames)],
+        ['span&']: [
+          {
+            visibility: 'hidden',
+            position: 'absolute',
+            right: 0,
+          },
+        ],
```

Comment:
FYI @czearing, you will need to do this TODO when merging #15216. I would normally wait until after your PR is merged for this change (and not change it myself), but it looks like your PR is being held back for now. It should look like this after the TODO:
```js
overflowMenuButton: [
  classNames.overflowMenuButton,
  ...getLinkStyles(props, classNames),
  {
    visibility: 'hidden',
    position: 'absolute',
    right: 0,
  },
],
```